stdlib/public/core/Runtime.swift.gyb

```diff
@@ -383,7 +386,8 @@ func _float${bits}ToString(_ value: Float${bits}, debug: Bool) -> String {
   var buffer = _Buffer32()
   return withUnsafeMutablePointer(to: &buffer) {
     (bufferPtr) in
-    let bufferUTF8Ptr = UnsafeMutablePointer<UTF8.CodeUnit>(bufferPtr)
+    let bufferUTF8Ptr = UnsafeMutableRawPointer(bufferPtr)
+      .bindMemory(to: UTF8.CodeUnit.self, capacity: 32)
```

Comment:
If changing the definitions of `_Buffer32` and `_Buffer72` to use `UInt8` instead of `UInt64` would help anything here, please feel free to change them!